src/notification.rs

```diff
@@ -449,16 +459,17 @@ impl Default for Notification {
     #[cfg(target_os="windows")]
     fn default() -> Notification {
         Notification {
-            appname:  exe_name(),
-            summary:  String::new(),
-            subtitle:  None,
-            body:     String::new(),
-            icon:     String::new(),
-            actions:  Vec::new(),
-            timeout:  Timeout::Default,
-            sound_name: Default::default(),
-            id:       None,
-            path_to_image: None
+            appname:       exe_name(),
```

Comment:
did you run rustfmt here?

Reply:
I manually updated these, forgot about rustfmt, oops

Reply:
running rustfmt changed too much the source code, so it'll leave this as is.